docs/tutorials/excel-tutorial-create-table.md

```diff
@@ -1,10 +1,10 @@
 ---
```

Comment:
Why is this file part of this PR?

Reply:
So I accidentally changed this to be a blank description, I believe in an earlier commit. I thought I'd fix. I can try to pull it out and not make it part of this PR if that is clearer (probably would be).

Reply:
Ah, okay. No worries, it can stay part of this PR.  Others will see this comment thread and understand why it's here.